Examples.xcodeproj/project.pbxproj

```diff
@@ -80,6 +80,10 @@
 		3EC92DB71E78C431001D0503 /* metro-line.geojson in Resources */ = {isa = PBXBuildFile; fileRef = 3EC92DB61E78C431001D0503 /* metro-line.geojson */; };
 		3ED403411E006B5200230C95 /* CameraFlyToExample.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3ED403401E006B5200230C95 /* CameraFlyToExample.swift */; };
 		3ED403481E006BE800230C95 /* CameraFlyToExample.m in Sources */ = {isa = PBXBuildFile; fileRef = 3ED403471E006BE800230C95 /* CameraFlyToExample.m */; };
+		589F5DE72355771E00548346 /* GradientGeoJSONLineExample.m in Sources */ = {isa = PBXBuildFile; fileRef = 589F5DE62355771E00548346 /* GradientGeoJSONLineExample.m */; };
+		589F5DEA2355782000548346 /* GradientGeoJSONLineExample.swift in Sources */ = {isa = PBXBuildFile; fileRef = 589F5DE92355782000548346 /* GradientGeoJSONLineExample.swift */; };
+		589F5DEC2355785500548346 /* mapbox_access_token in Resources */ = {isa = PBXBuildFile; fileRef = 589F5DEB2355785500548346 /* mapbox_access_token */; };
```

Comment:
It looks like your access token was committed. You'll want to remove this and then revoke the token that was being used. Let me know if you have questions about this.